01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,11 @@
+for (let i = 1; i <= 20; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
```

Comment:
基本的に厳密等価演算子 (===) を使った方が良さそうです ☺️ 
※ ここ以外も同様に

ref: https://jsprimer.net/basic/implicit-coercion/#implicit-coercion

```suggestion
    if (i % 3 === 0 && i % 5 === 0) {
```